.github/workflows/web-release.yml

```diff
@@ -50,18 +49,31 @@ jobs:
     - name: Create releases directory
       run: mkdir -vp web/explorer/releases
 
-    - name: Move archive release to releases folder
+    - name: Move release to releases folder
       run: mv web/explorer/${{ env.RELEASE_NAME }}.zip web/explorer/releases
 
-    - name: Commit and push release
+    - name: Compute release SHA256 hash
       run: |
-        git config --local user.email "capa-dev@mandiant.com"
-        git config --local user.name "Capa Bot"
-        git add -f web/explorer/releases/${{ env.RELEASE_NAME }}.zip
-        git commit -m "explorer web: add release ${{ env.RELEASE_NAME }}"
-        git push
+        echo "RELEASE_SHA256=$(sha256sum web/explorer/releases/${{ env.RELEASE_NAME }}.zip | awk '{print $1}')" >> $GITHUB_ENV
```

Comment:
this should be the hash of the inner HTML file

Reply:
for consistency with VM-Packages, I'd recommend keeping the hash of the zip file releases, since that's what is declared in the nuspec definitions.

Reply:
oh yes, I thought it's the inner file, but was aiming for consistency with VM-Packages here